src/include/daos_srv/control.h

```diff
@@ -32,6 +32,15 @@
 #include <stdbool.h>
 #include <assert.h>
 
+enum {
+	/* Device is plugged */
+	NVME_DEV_FL_PLUGGED	= 0x1,
```

Comment:
these can be used to decode bdi_flags in the drpc list-devices caller in src/control/server package